upf/upf_api.c

```diff
@@ -845,6 +845,54 @@ vl_api_upf_pfcp_policer_show_t_handler (vl_api_upf_pfcp_policer_show_t * mp)
   vl_api_send_msg (reg, (u8 *) rmp);
 }
 
+/* API message handler */
+static void
+vl_api_upf_pfcp_heartbeats_set_t_handler (vl_api_upf_pfcp_heartbeats_set_t *
+					  mp)
+{
+  vl_api_upf_pfcp_heartbeats_set_reply_t *rmp = NULL;
+  u32 retries, timeout = 0;
```

Comment:
What purpose does `timeout = 0` assignment serve here?

Reply:
changed to `~0`